test/JDBC/upgrade/13_6/schedule

```diff
@@ -233,7 +233,7 @@ sys-schema-name
 sys-schemas
 sys-schemas-dep
 sys-selective_xml_index_paths
-sys-server_principals
+sys-server_principals_before-15_5
```

Comment:
Name is incorrect, so the tests are failing. 
It should be `sys-server_principals-before-15_5`